Packs/TeamCymru/Integrations/TeamCymru/README.md

```diff
@@ -10,7 +10,9 @@ This integration was integrated and tested with version 1.0 of TeamCymru
     | **Parameter** | **Required** |
     | --- | --- |
     | Use system proxy settings | False |
-    | Proxy URL | Supports socks4/socks5/http connect proxies \(e.g., socks5h://host:1080\). | False |    
+    | Proxy URL | Supports socks4/socks5/http connect proxies \(e.g., socks5h://host:1080\). | False | 
```

Comment:
I think we are also missing the reputation as output in both commands

Reply:
already there, check lines 48, 135.